api/src/main/java/org/openmrs/Provider.java

```diff
@@ -92,6 +96,48 @@ public String getIdentifier() {
 		return identifier;
 	}
 	
+	/**
+	 * Sets the role concept
+	 * 
+	 * @since 2.2
+	 * @param role the role to set
+	 *                
```

Comment:
To be consistent, can we remove this line?

Reply:
Actually this is the right thing to do otherwise you get IDE warnings, plus there domain objects where this s done